src/App.js

```diff
@@ -1,18 +1,22 @@
+import React from 'react';
 import './App.css';
 import { BrowserRouter, Routes, NavLink, Route } from 'react-router-dom';
-import List from './components/List.js';
-import AddItem from './components/AddItem.js';
+import Home from './components/Home.jsx';
+import List from './components/List.jsx';
+import AddItem from './components/AddItem.jsx';
 
 function App() {
   return (
     <div className="App">
       <BrowserRouter>
         <Routes>
-          <Route path="/" element={<List />} />
-          <Route path="/additem" element={<AddItem />} />
+          <Route exact path="/" element={<Home />} />
+          <Route path="/list" element={<List />} />
+          <Route path="/addItem" element={<AddItem />} />
         </Routes>
+      </BrowserRouter>
 
-        <nav className="link-div">
+      {/* <nav className="link-div">
           <NavLink
             to="/"
             style={({ isActive }) =>
```

Comment:
Why did you decide to comment out the NavLink code? Was it not working properly with it? With this code removed it doesn't look like we are able to navigate to the AddItem component. How were you accessing that code?

Reply:
We had commented out the code temporarily because it wasn't working for us at the time. We tried to move this conditional rendering over to exist only in the 'list' and 'additem' views but unfortunately were unable to implement that. It currently remains on the app.js file and renders on all routes including 'home' which we will need to fix at some point, but it's working again.

Reply:
Sounds good! Thanks for clarifying that and letting us know that it will be something that we'll want to look at and fix later.